app/license.go

```diff
@@ -245,3 +257,66 @@ func (s *Server) RequestTrialLicense(trialRequest *model.TrialLicenseRequest) *m
 
 	return nil
 }
+
+// GenerateRenewalToken returns the current active token or generate a new one if
+// the current active one has expired
+func (s *Server) GenerateRenewalToken(expiration time.Duration) (string, *model.AppError) {
```

Comment:
Is there anything here we should include in the enterprise repo instead?

Reply:
No need. I've talked in the server guild and they want the license code in the team edition unless there is a good reason for it and security by obscurity is not one